frontend/web/components/tags/Tag.tsx

```diff
@@ -66,6 +66,14 @@ const Tag: FC<TagType> = ({
     )
   }
 
+  // Hide unhealthy tags if feature is disabled
+  if (
+    !Utils.getFlagsmithHasFeature('feature_health') &&
+    tag.type === 'UNHEALTHY'
+  ) {
+    return null
+  }
+
```

Comment:
Hiding feature behind feature flag just in case a feature gets tagged as unhealthy and that functionality is not yet available somehow

Reply:
Good call, it should, I'll update that in a minute